Makefile

```diff
@@ -104,13 +106,25 @@ CSI_BIN_SRCS += $(addsuffix /*.go,$(shell go list -f '{{ join .Deps "\n" }}' ./c
 export CSI_BIN_SRCS
 endif
 $(CSI_BIN): $(CSI_BIN_SRCS)
-	CGO_ENABLED=0 GOOS=$(GOOS) GOARCH=$(GOARCH) go build -ldflags '$(LDFLAGS_CSI)' -o $(CSI_BIN_LINUX) $<
+	CGO_ENABLED=0 GOOS=$(GOOS) GOARCH=$(GOARCH) go build $(GOFLAGS_VENDOR) -ldflags '$(LDFLAGS_CSI)' -o $(CSI_BIN_LINUX) $<
 	@touch $@
 
 $(CSI_BIN_WINDOWS): $(CSI_BIN_SRCS)
-	CGO_ENABLED=0 GOOS=windows GOARCH=$(GOARCH) go build -ldflags '$(LDFLAGS_CSI)' -o $(CSI_BIN_WINDOWS) $<
+	CGO_ENABLED=0 GOOS=windows GOARCH=$(GOARCH) go build $(GOFLAGS_VENDOR) -ldflags '$(LDFLAGS_CSI)' -o $(CSI_BIN_WINDOWS) $<
 	@touch $@
 
+# The cnsctl binary.
```

Comment:
You add all this clause (L116-127) only to remove it in the next patch of this PR. Also, that patch removing it also removes extra empty line. This way we'll end up in extra discrepancy between openshift and upstream repos. I think we have to modify `Makefile` only once in this PR (i.e. squash these two changes).

Reply:
Hmm, so upstream used to carry this line and then they removed it. So if you see current master of this fork, it has those cnsctl changes, but 3.0 tag removes it and that is why you see two commits. We used to build `cnsctl` binary and now we don't..

Reply:
I can see a case of squashing the commits though. Lesser "out of band" commits, we have to deal with during the rebase the better..